src/modules/gtplogger.js

```diff
@@ -0,0 +1,195 @@
+const {remote} = require('electron')
+const fs = require('fs');
+const setting = remote.require('./setting')
+const dialog = require('./dialog')
+const winston = require('winston');
+const path = require('path')
+
+let gtplogger = null
```

Comment:
Maybe call this variable something else to avoid confusion. `winstonLogger`, or simply `logger`? On another note, can't we call `winston.createLogger` right here?

Reply:
Yep. Done.